src/Tgstation.Server.Host/Core/SetupWizard.cs

```diff
@@ -166,8 +167,18 @@ async Task<DatabaseConfiguration> ConfigureDatabase(CancellationToken cancellati
 				await console.WriteAsync(null, true, cancellationToken).ConfigureAwait(false);
 				await console.WriteAsync("Enter the server's address and port (blank for local): ", false, cancellationToken).ConfigureAwait(false);
 				var serverAddress = await console.ReadLineAsync(false, cancellationToken).ConfigureAwait(false);
+				var serverPort = 3306U;
 				if (String.IsNullOrWhiteSpace(serverAddress))
 					serverAddress = null;
+				else
```

Comment:
```suggestion
				else if (databaseConfiguration.DatabaseType != DatabaseType.SqlServer)
```
The ConnectionStringBuilder for SQL Server includes its port in the `DataSource` property.

Reply:
Or maybe rename `serverPort` to `mySQLServerPort`